layouts/partials/cards/encounters/fight_card.html

```diff
@@ -0,0 +1,21 @@
+<div class="relative mb-8 overflow-hidden">
+  <a class="block bg-card-light" {{ if (eq .Params.coming_soon false) }}href="{{ .Permalink }}"{{ end }}>
+    {{ if eq .Params.spoiler true}}
+      <img class="block object-cover w-full filter blur-md" src="{{ .Params.tier_image}}" alt="" />
+    {{ else }}
+      <img class="block object-cover w-full" src="{{ .Params.tier_image}}" alt="" />
+    {{ end }}
+    <div class="bg-card-light">
+      <h2 class="flex items-center text-md md:text-md uppercase px-4 font-semibold tracking-wider py-2 mb-0">{{ .Params.fight_title }}</h2>
```

Comment:
I actually think this might be better to do as just `{{ .Title }}` so we get a meaningful name instead of `e12s` and instead get the good fight name.

Reply:
Resolved!